src/processor/intents/creeps/tick.js

```diff
@@ -89,16 +89,7 @@ module.exports = function(object, roomObjects, roomTerrain, bulk, bulkUsers, roo
             }
 
             if(gameTime >= object.ageTime-1) {
-
-                C.RESOURCES_ALL.forEach(resourceType => {
-                    var amount = object[resourceType];
-                    if (amount) {
-                        require('./_create-energy')(object.x, object.y, object.room, amount, roomObjects, bulk, resourceType, object.dropToContainer);
-                    }
-                });
-
-                bulk.remove(object._id);
-                delete roomObjects[object._id];
+                require('./_die')(object, roomObjects, bulk, stats, undefined, gameTime);
```

Comment:
Right now this will add to the `creepsLost` stat, which isn't exactly what you want.
Using `undefined` instead of `stats` will fix this already, but maybe it's better to add options to disable this behaviour?

Reply:
hmm, IIRC, somewhere else it also passes undefined as stats, updating now

Reply:
its spawn recycle-creep where it passes undefined